reference/5.1/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Methods.md

```diff
@@ -96,6 +96,34 @@ For more information, see [about_Automatic_Variables][15].
 > block isn't defined, record-by-record processing will fail. In this case,
 > your function will only execute once, regardless of the input.
 
+When you create a function accepts pipeline input and uses `CmdletBinding`, the
```

Comment:
I appreciate the addition.

A few suggested tweaks:

* "that" is missing before "accepts" in the first sentence.

* Unlike stated, the sample function doesn't actually use `CmdletBinding` - but it doesn't need to, because the use of a `[Parameter()]` attribute _alone_ makes a script or function an advanced one (which is the common thread here).

* Conversely, however, a function/script with `CmdletBinding` that wants to accept pipeline input _must_ also declare a parameter with `[Parameter(ValueFromPipeline)]` (optionally complemented by  `[Parameter(ValueFromPipelineByPropertyName)]` parameters).